repometadata/templates/.github/workflows/workflow.yml.jinja2

```diff
@@ -28,9 +28,9 @@ jobs:
       uses: actions/checkout@v4
 
     - name: Set up conda
-      uses: conda-incubator/setup-miniconda@v4
+      uses: conda-incubator/setup-miniconda@v3
```

Comment:
Why can't we use the latest version?

Reply:
We probably could. I had those commits lying around for a while and guess v4 was just not released yet when I did the work. I'll just leave this open for now and update when I find time.